src/ec/packages/ec-component-button/__snapshots__/button.test.js.snap

```diff
@@ -1,87 +1,147 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`EC - Button CTA button - icon after renders correctly 1`] = `
-"<button class=\\"ecl-button ecl-button--call\\" type=\\"submit\\">
-  <span class=\\"ecl-button__container\\">
-    <span class=\\"ecl-button__label\\">CTA Button with icon</span>
-
+<button
+  class="ecl-button ecl-button--call"
+  type="submit"
+>
+  <span
+    class="ecl-button__container"
+  >
+    <span
+      class="ecl-button__label"
+    >
+      CTA Button with icon
+    </span>
     <svg
-      class=\\"ecl-icon ecl-icon--fluid ecl-button__icon ecl-button__icon--after\\"
-      focusable=\\"false\\"
-      aria-hidden=\\"true\\"
+      aria-hidden="true"
+      class="ecl-icon ecl-icon--fluid ecl-button__icon ecl-button__icon--after"
+      focusable="false"
     >
-      <use xlink:href=\\"static/icons.svg#ui--corner-arrow\\"></use>
+      <use
+        xlink:href="static/icons.svg#ui--corner-arrow"
+      />
     </svg>
   </span>
 </button>
-"
 `;
 
 exports[`EC - Button CTA button - icon before renders correctly 1`] = `
-"<button class=\\"ecl-button ecl-button--call\\" type=\\"submit\\">
-  <span class=\\"ecl-button__container\\">
+<button
+  class="ecl-button ecl-button--call"
+  type="submit"
+>
+  <span
+    class="ecl-button__container"
+  >
     <svg
-      class=\\"ecl-icon ecl-icon--fluid ecl-icon--rotate-90 ecl-button__icon ecl-button__icon--before\\"
-      focusable=\\"false\\"
-      aria-hidden=\\"true\\"
+      aria-hidden="true"
+      class="ecl-icon ecl-icon--fluid ecl-icon--rotate-90 ecl-button__icon ecl-button__icon--before"
+      focusable="false"
     >
-      <use xlink:href=\\"static/icons.svg#ui--corner-arrow\\"></use>
+      <use
+        xlink:href="static/icons.svg#ui--corner-arrow"
+      />
     </svg>
-    <span class=\\"ecl-button__label\\">Call to action button</span>
+    <span
+      class="ecl-button__label"
+    >
+      Call to action button
+    </span>
   </span>
 </button>
-"
 `;
 
 exports[`EC - Button CTA renders correctly 1`] = `
-"<button class=\\"ecl-button ecl-button--call\\" type=\\"submit\\">
-  <span class=\\"ecl-button__container\\">
-    <span class=\\"ecl-button__label\\">Call to action button</span>
-
+<button
+  class="ecl-button ecl-button--call"
+  type="submit"
+>
+  <span
+    class="ecl-button__container"
+  >
+    <span
+      class="ecl-button__label"
+    >
+      Call to action button
```

Comment:
this can be tricky... if the output of the twig template was `<span class="ecl-button__label"> Call to action button </span>`, then the snapshot would add a space before and after `Call to action`. It's not easy to catch it